spec/requests/authentication_spec.rb

```diff
@@ -115,7 +115,12 @@
       get api_entrypoint_url, :params => { :attributes => "authorization" }
 
       expect(response).to have_http_status(:ok)
-      expected = {"authorization" => hash_including("product_features")}
+      expected = {
+        "authorization" => hash_including("product_features"),
+        "identity"      => hash_including("miq_groups" => a_collection_including(
+          hash_including("href" => api_users_url(nil, @user.miq_groups.first))
+        ))
+      }
```

Comment:
I think the miq_groups in identity is returned with all calls  (i.e. without attributes=authorization), in which case if you can update the description and move the test here to the regular entrypoint (not authorization).